### PR TITLE
Updated Orca4 SDF file to use BlueROV2 models

### PR DESCRIPTION
**Pull request by ssicari**
I moved over the meshes folder and edited the Orca4 model.sdf file to incorporate the full model.

### CODE REVIEW

**Comment by clydemcqueen** [comment]
Very cool! I ran this and it looks and works great.

I'd like to see the changes made to the model.sdf.in and the generator script (generate_model.py) such that when you run the script you get the correct results in model.sdf. That way it is easy to change the generator script to experiment with different parameters.

See the changes made to bluerov2_ignition for an example: https://github.com/clydemcqueen/bluerov2_ignition/pull/6/files

**Comment by ssicari** [comment]
> Very cool! I ran this and it looks and works great.
> 
> I'd like to see the changes made to the model.sdf.in and the generator script (generate_model.py) such that when you run the script you get the correct results in model.sdf. That way it is easy to change the generator script to experiment with different parameters.
> 
> See the changes made to bluerov2_ignition for an example: https://github.com/clydemcqueen/bluerov2_ignition/pull/6/files

I made the changes based on the link example. I am a little confused on what exactly those changes are for and what exactly the generator script and model.sdf.in do. If you could give me a more in explanation that would be appreciated. Let me know if I need to make more changes, thank you!

**Comment by clydemcqueen** [comment]
I use generator.py to re-write orca4/model.sdf from orca4/model.sdf.in, replacing strings like "@camera_y" with the result of the expression "stereo_baseline / 2". This makes it easy to build models with different camera placements and so forth.

To modify the model, update model.sdf.in and then run generate.py:
~~~
cd orca_description
scripts/generate_model.py models/orca4/model.sdf.in models/orca4/model.sdf 0 
~~~

File: orca_description/models/orca4/model.sdf.in [hunk 1]
```diff
@@ -514,114 +417,5 @@
 
     </plugin>
 
-    <!-- Left camera link -->
```
Comment (clydemcqueen) [inline orca_description/models/orca4/model.sdf.in:517]: The camera links, joints and sensors are required. Please restore.

File: orca_description/scripts/generate_model.py [hunk 2]
```diff
@@ -1,34 +1,12 @@
 #!/usr/bin/env python3
 
-# MIT License
```
Comment (clydemcqueen) [inline orca_description/scripts/generate_model.py:3]: Please restore the license.

File: orca_description/scripts/generate_model.py [hunk 3]
```diff
@@ -1,34 +1,12 @@
...
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-# SOFTWARE.
-
 """
 Generate the model.sdf file by substituting strings of the form "@foo" with calculated values
 
 The SDF file uses the ArduPilotPlugin COMMAND control method; this sends commands to a specified
 ign-transport topic rather than directly controlling a joint.
 
-We use the COMMAND method to send commands to the Gazebo Sim ThrusterPlugin. The ThrusterPlugin
```
Comment (clydemcqueen) [inline orca_description/scripts/generate_model.py:31]: The new Gazebo is called "Gazebo Sim" to distinguish it from the old Gazebo.

File: orca_description/scripts/generate_model.py [hunk 4]
```diff
@@ -123,20 +100,6 @@
 thruster5_topic = "/model/orca4/joint/thruster5_joint/cmd_"
 thruster6_topic = "/model/orca4/joint/thruster6_joint/cmd_"
 
-# Stereo camera
```
Comment (clydemcqueen) [inline orca_description/scripts/generate_model.py:126]: The camera section is required.

**Review by clydemcqueen — CHANGES_REQUESTED** [review]

**Comment by ssicari** [comment]
> I use generator.py to re-write orca4/model.sdf from orca4/model.sdf.in, replacing strings like "@camera_y" with the result of the expression "stereo_baseline / 2". This makes it easy to build models with different camera placements and so forth.
> 
> To modify the model, update model.sdf.in and then run generate.py:
> 
> ```
> cd orca_description
> scripts/generate_model.py models/orca4/model.sdf.in models/orca4/model.sdf 0 
> ```

I made the changes you requested, that was my fault when I was copying and pasting I must have not been paying close enough attention to differences between the files. I appreciate the responses it has cleared up some confusion as well.

**Review by clydemcqueen — COMMENTED** [review]
Yes, that is how the generator works. I should just remove orca4/model.sdf from the repo and generate it from orca4/model.sdf.in during the build step. I'll do this in another PR.

This is useful for avoiding copy/paste errors while experimenting with the model. It is easy to change the max thrust or mass or stereo baseline, then re-run the generator and the sdf file is updated.

Two typos, then this is done!

Thanks for this great contribution to orca4!

**Comment by ssicari** [comment]
> Yes, that is how the generator works. I should just remove orca4/model.sdf from the repo and generate it from orca4/model.sdf.in during the build step. I'll do this in another PR.
> 
> This is useful for avoiding copy/paste errors while experimenting with the model. It is easy to change the max thrust or mass or stereo baseline, then re-run the generator and the sdf file is updated.
> 
> Two typos, then this is done!
> 
> Thanks for this great contribution to orca4!

Should be all set now, thank you!

**Review by clydemcqueen — APPROVED** [review]

**Pull request merged**